### DISCUSSION TITLE
Relation between "gpgpu-sim" and "accel-sim", and "gpgpu-sim_distribution" Git branches

### DISCUSSION THREAD

**Discussion by tschwinge** [accel-sim/accel-sim-framework#217] [General] [answered]
I'm unclear on the relation between <https://github.com/gpgpu-sim/> and <https://github.com/accel-sim/>, and their *gpgpu-sim_distribution* Git branches: <https://github.com/gpgpu-sim/gpgpu-sim_distribution/branches> and <https://github.com/accel-sim/gpgpu-sim_distribution/branches>.  I understand *accel-sim* is a successor to *gpgpu-sim*?  Does it mean that the latter is effectively frozen (and should be marked as such, or its Git repositories be archived, and visitors redirected to <https://github.com/accel-sim/>)?

Current development seems to happen only on the *dev* branch (not *master* branch) in *accel-sim*: <https://github.com/accel-sim/gpgpu-sim_distribution/tree/dev>.  Is that where any Pull Requests should be filed against?

**Comment by JRPan** [comment, marked as answer]
Thank you for the question. 

1. Accel-Sim does not necessarily be a successor to gpgpu-sim. Even though Accel-Sim (SASS-mode) can achieve higher accuracy and shorter simulation time compared to gpgpu-sim, many people still use the latter. Any gpgpu-sim specific question should be directed to https://github.com/gpgpu-sim/. 
2. Accel-Sim's backend is gpgpu-sim, which is here https://github.com/accel-sim/gpgpu-sim_distribution. Over a period of time, these changes will be merged to the gpgpu-sim which is maintained by folks at UBC https://github.com/gpgpu-sim/. We actively test our branch of gpgpu-sim to make sure it's compatible with the legacy PTX-mode.
3. At Accel-Sim, all features and new PRs should be filed to dev branch. Then after a certain period, we make the next release, and the dev will be merged to release (the master branch) with all the changes. Right now, the release is actually **v1.2.0+** (as of 05/23/2023) as we merged dev once into release after v1.2.0 but did not announce a release. We will do our next major release after all current issues and PRs are resolved. Hopefully in the next month. 

I think lots of fellows may have the same question. So I'll pin this discussion. Thank you!
